packages/amplify_authenticator/example/lib/main.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+import 'package:amplify_flutter/amplify.dart';
+import 'package:flutter/material.dart';
+import 'package:amplify_authenticator/amplify_authenticator.dart';
+
+import 'amplifyconfiguration.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: MyHomePage(title: 'Flutter Demo Home Page'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  int _counter = 0;
+
+  void _incrementCounter() {
+    setState(() {
+      _counter++;
+    });
+  }
+
+  @override
+  void initState() {
+    super.initState();
+
+    _amplifyConfig();
+  }
+
+// Amplify Configuration
+  void _amplifyConfig() async {
+    try {
+      await Amplify.addPlugin(AmplifyAuthCognito());
+      await Amplify.configure(amplifyconfig);
+    } catch (e) {
+      print(e);
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Authenticator(
+      signUpForm: SignUpForm(
+        formFields: FormFields(children: [
+          SignUpFormField(
+            type: 'username',
+            title: "Email or Phone Number*",
+            hintText: "Enter your username",
+          ),
+          SignUpFormField(
+              title: 'Password*',
+              hintText: "Enter your password",
+              type: "password"),
+          SignUpFormField(
```

Comment:
I think on the example app we can stick to username, email and password.

Reply:
Yes. I was testing it out with most of the custom form fields and forgot to remove them. The default form only includes username, email, password and phone number, and there is no need to pass any form field arguments in the "Authenticator" wrapper.